### PR TITLE
POM contents

### PR DESCRIPTION
**Pull request by timowest**
Related to https://github.com/querydsl/querydsl/issues/1132

### CODE REVIEW

**Comment by johnktims** [comment]
Should this include license information?

**Comment by timowest** [comment]
Sounds good, added.

**Comment by Shredder121** [comment]
The members of The Querydsl Team could be added under `<developers>`, right beneath the `<organization>` element.
This consolidates the developers in the root querydsl pom.
What do you think?

**Comment by timowest** [comment]
Makes sense, included in the newest version.

**Comment by Shredder121** [comment]
`<description>` inherited and `<scm>` incorrectly inherited, best to override these.

**Comment by johnktims** [comment]
This looks fine to me now.

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+  <organization>
+    <name>Querydsl</name>
+    <url>http://www.querydsl.com</url>
+  </organization>
+
+  <developers>
+    <developer>
+      <id>timowest</id>
+      <name>Timo Westkämper</name>
+      <organization>Mysema Ltd</organization>
```
Comment (Shredder121) [inline pom.xml]: @timowest Do we make the Querydsl Team members organization neutral? Or do you want this included?

Comment (timowest) [inline pom.xml]: Organization neutrality makes sense. I'll change it.

**Pull request merged**